docs/data/sdks/java/index.md

```diff
@@ -204,34 +204,34 @@ client.logEvent(event);
     If Joe is in 'orgId' '10', then the `groupName` would be '10':
 
     ```java
-    Event event = new Event("$identify", "test_user_id");
+    Event event = new Event("$identify");
```

Comment:
For setGroup we wil do 2 things:
- We set the groups
- We set the userProperties

[Reference](https://github.com/amplitude/Amplitude-Kotlin/blob/main/core/src/main/java/com/amplitude/core/Amplitude.kt#L307-L334).

Reply:
Is this the difference between server sdk and client sdk?

Reply:
I referenced [Python](https://github.com/amplitude/Amplitude-Python/blob/576bdea85d202f97784c79a4e1833dad50890e82/src/amplitude/client.py#L121-L133) where we only set `event.groups` but not `event.userProperties`. But Node does set both of them

Reply:
It seems to be a bug that Python SDK doesn't set the user to be the group. For example, I tested that `example_user` is not set to group `Infra`.
```
amp_client.set_group("Eng Team", "Infra", EventOptions(user_id="example_user"))
```

Reply:
Create a [ticket](https://amplitude.atlassian.net/browse/AMP-83718) for Python and fixed here